docs/actionable-messages/auto-invoke.md

```diff
@@ -18,7 +18,7 @@ Actionable messages allow users to take quick actions on an email message, often
 
 ## Using autoInvokeAction
 
-In order to use this feature, your card must use the [Adaptive Card](adaptive-card.md) format. The [autoInvokeAction](adaptive-card.md#additional-properties-on-the-adaptivecard-type) property is an Outlook-specific property added to the AdaptiveCard type. The value of this property is an [Action.Http](adaptive-card.md#actionhttp) action with the `method` set to `POST`. The `url` property specifies a Web API endpoint in your service that will provide the updated Adaptive Card payload.
+In order to use this feature, your card must use the [Adaptive Card](adaptive-card.md) format. The [autoInvokeAction](adaptive-card.md#additional-properties-on-the-adaptivecard-type) property is an Outlook-specific property added to the AdaptiveCard type. The value of this property is an [Action.Http](adaptive-card.md#actionhttp) action with the `method` set to `POST`. The `url` property specifies a Web API endpoint in your service that will provide the updated Adaptive Card payload. You also need to provide `CARD-UPDATE-IN-BODY` header in response with value `true` in order for your response to be accepted.
```

Comment:
Could you move this new sentence to the **Implementing the Web API** section? Specifically, there's a list of requirements:

> The endpoint for an autoInvokeAction must also meet the following additional requirements.
> 
> - Requests must return within 2 seconds.
> - Requests that take longer will be ignored by the client, and the original card will continue to display. The message will still be updated on the server.
> - Successful responses should include an Adaptive Card JSON payload.

This would go well as an addition to the third bullet, something like "Successful responses should include a `CARD-UPDATE-IN-BODY` header and an Adaptive Card JSON payload."

Reply:
Done!